components/gitpod-protocol/src/workspace-instance.ts

```diff
@@ -62,6 +62,13 @@ export interface WorkspaceInstance {
      * resources that are provided to the workspace.
      */
     workspaceClass?: string;
+
+    /**
+     * Identifies the team to which this instance's runtime should be attributed to
+     * (e.g. for usage analytics or billing purposes).
+     * If unset, the usage should be attributed to the workspace's owner (ws.ownerId).
+     */
+    attributedTeamId?: string;
```

Comment:
Rather than leaving this unset in this case, should we explicitly fill in the ownerId?

Reply:
I believe leaving it unset makes more sense:
- It can only be a `teamId` (the billing is either delegated to a specific team, or it falls back to you -- delegating to a different user does not make sense, right? 🤔)
- Setting it to possibly a `userId` makes the design more complex (need a second column `attributedUserId`, or maybe a prefix to the ID e.g. `team-1324` vs `user-1234`)
- The team attribute link goes through instance > workspace > project > team (any of the links can change so persisting the endpoint makes sense). User attribution is more direct (instance > workspace > owner) -- I don't think we can even get to a state where we don't know which user an instance belongs to.

Reply:
> delegating to a different user would not make sense, right?
Yes, that's my understanding too.